src/it/fails-on-broken-xmir/README.md

```diff
@@ -0,0 +1,11 @@
+# Generates SODG files
```

Comment:
@h1alexbel [todo] It's an awesome test. That said, it checks only, `failOnXmirErrors = true` (default value), maybe we need to add one more test, that would explicitly set `failOnXmirErrors = false`?